src/components/include/cos_kernel_api.h

```diff
@@ -21,6 +21,10 @@ typedef capid_t compcap_t;
 typedef capid_t captblcap_t;
 typedef capid_t pgtblcap_t;
 
+/* Tcap typedefs */
+typedef s64_t tcap_res_t;
+typedef u64_t tcap_prio_t;
```

Comment:
Redundant with the defs in the kernel.

Should be in kernel/include/shared/cos_types.h